src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -0,0 +1,269 @@
+import * as clipboard from 'clipboard-polyfill/text'
+import _ from 'lodash/fp'
+import { useEffect, useState } from 'react'
+import { div, h, span, table, tbody, td, tr } from 'react-hyperscript-helpers'
+import ReactJson from 'react-json-view'
+import * as breadcrumbs from 'src/components/breadcrumbs'
+import Collapse from 'src/components/Collapse'
+import { Link } from 'src/components/common'
+import { centeredSpinner, icon } from 'src/components/icons'
+import {
+  collapseCromwellExecutionStatus,
+  failedIcon,
+  makeSection,
+  makeStatusLine,
+  runningIcon,
+  statusIcon, submittedIcon,
+  successIcon, unknownIcon
+} from 'src/components/job-common'
+import UriViewer from 'src/components/UriViewer'
+import WDLViewer from 'src/components/WDLViewer'
+import { Ajax } from 'src/libs/ajax'
+import { bucketBrowserUrl } from 'src/libs/auth'
+import { getConfig } from 'src/libs/config'
+import { withErrorReporting } from 'src/libs/error'
+import * as Nav from 'src/libs/nav'
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+import { wrapWorkspace } from 'src/pages/workspaces/workspace/WorkspaceContainer'
+
+
+const styles = {
+  sectionTableLabel: { paddingRight: '0.6rem', fontWeight: 600 }
+}
+
+const groupCallStatuses = callsObject => {
+  const statusCounts = {}
+  for (const callname in callsObject) {
+    for (const attempt in callsObject[callname]) {
+      const executionStatus = collapseCromwellExecutionStatus(callsObject[callname][attempt].executionStatus)
+      statusCounts[executionStatus] = (statusCounts[executionStatus] || 0) + 1
+    }
+  }
+  return statusCounts
+}
+
+const statusCell = workflowObject => {
+  const { succeeded, failed, running, submitted, ...others } = groupCallStatuses(workflowObject.calls)
+  console.log(others)
+
+  return div([
+    table({ style: { margin: '0.5rem' } }, [
+      tbody({}, [
+        submitted ? tr({}, [
+          td(styles.statusDetailCell, [submittedIcon()]),
+          td(['Submitted']),
+          td(styles.statusDetailCell, [submitted])
+        ]) : undefined,
+        running ? tr({}, [
+          td(styles.statusDetailCell, [runningIcon()]),
+          td(['Running']),
+          td(styles.statusDetailCell, [running])
+        ]): undefined,
+        succeeded ? tr({}, [
+          td(styles.statusDetailCell, [successIcon()]),
+          td(['Succeeded']),
+          td(styles.statusDetailCell, [succeeded])
+        ]): undefined,
+        failed ? tr({}, [
+          td(styles.statusDetailCell, [failedIcon()]),
+          td(['Failed']),
+          td(styles.statusDetailCell, [failed])
+        ]): undefined,
+        _.map(other => tr({}, [
+          td(styles.statusDetailCell, [unknownIcon()]),
+          td([other]),
+          td(styles.statusDetailCell, [others[other]])
+        ]), Object.keys(others))
+      ].filter(element => element !== undefined))
+    ])
+
+  ])
```

Comment:
There are a few things to talk about here:
- using the native js `.filter` and `Object.keys` rather than lodash
- unnecessary wrapping div
- ternaries rather than `&&`
- unneeded empty props for `tr`'s
- returning an array as a child, which makes React sad (see the console on the page)

Probably one to talk through in person, again ❤️

Reply:
Mostly resolved